templates/http/default.conf.j2

```diff
@@ -454,7 +454,7 @@ server {
 
     }
 {% endfor %}
-{% if item.value.servers[server].web_server.http_demo_conf %}
+{% if item.value.servers[server].web_server.http_demo_conf is defined %}
```

Comment:
You'll still need to add a simple boolean check, otherwise the block will always get imported.
```suggestion
{% if item.value.servers[server].web_server.http_demo_conf is defined and item.value.servers[server].web_server.http_demo_conf %}
```